earth_enterprise/src/fusion/autoingest/sysman/InsetInfo.cpp

```diff
@@ -330,6 +331,31 @@ void CalcPacketGenInfo<InsetInfo<MercatorRasterProductAssetVersion> > (
     const bool             is_overlay_terrain_proj,
     const uint32           overlay_terrain_resources_min_level);
 
+template <typename ProductAssetVersion>
+void OverlapCalculator<ProductAssetVersion>::CalculateOverlap()
+{
+    if (env.type == AssetDefs::Imagery)
+    {
+        FindNeededImageryInsets(env.gencov,
+                                env.insets,
+                                env.numInsets,
+                                neededIndexes,
+                                env.beginMinifyLevel,
+                                env.endMinifyLevel);
+    }
+    else
+    {
+        FindNeededImageryInsets(env.gencov,
```

Comment:
Shouldn't this second call be `FindNeededTerrainInsets`?

Reply:
fat fingered. fixed

Reply:
I'm not sure what happened but this still calls the imagery version, not the terrain version.